README.md

```diff
@@ -1409,6 +1409,7 @@ _Related: [Wikis](#wikis)_
 - [Fugacious](https://fugacio.us) - Open source short-term secure messaging (OSSSM). ([Source Code](https://github.com/fugacious/fugacious)) `CC0-1.0` `Ruby`
 - [Hastebin](https://haste.zneix.eu/about.md) - Open source pastebin. (This is a fork with extended maintenance). ([Demo](https://haste.zneix.eu), [Source Code](https://github.com/zneix/haste-server)) `MIT` `Nodejs`
 - [LogPaste](https://github.com/mtlynch/logpaste) - Minimal pastebin web app that's easy to self-host and persists data to any S3-compatible backend. ([Demo](https://logpaste.com/)) `MIT` `Go`
+- [MicroBin](https://github.com/szabodanika/microbin) - A tiny, configurable, self-contained PasteBin and URL shortener written in Rust. ([Source Code](https://github.com/szabodanika/microbin)) `BSD-3-Clause` `Rust`
```

Comment:
Great work adding your contribution!

Two links to the same repo is not necessary. You should remove the second ([Source Code]()) link.